### DISCUSSION TITLE
working in development but not in production

### DISCUSSION THREAD

**Discussion by ahmedsamirdev** [tailwindlabs/tailwindcss#4815] [Help]
I have problem .. this in dev mode (nextjs)
![3](https://user-images.githubusercontent.com/16477954/123513373-3a31b100-d68d-11eb-86bc-577e0c3b612e.png)
and this in production and after pushing the code successfully
![unknown](https://user-images.githubusercontent.com/16477954/123513374-3e5dce80-d68d-11eb-885e-f02f96007610.png)

also "(Disclosure) from @headlessui/react" doesn't work ..!! although it works good in dev mode but not after in production
what could be the reason?

**Comment by ahmedsamirdev** [comment]
posted in headlessui instead

**Comment by VicGUTT** [comment]
Hey !

> what could be the reason?

The only reason I can think of is your templates are either not being purged correctly or them not being _[purge friendly](https://tailwindcss.com/docs/optimizing-for-production#writing-purgeable-html)_.

In any case, your description does not provide enough details to go off of. So it could really be a number of things.
